manifests/install.pp

```diff
@@ -1,8 +1,11 @@
+# == Class: check_mk::install
```

Comment:
Can we use puppet-strings style docs instead?

Reply:
Converted, however the check_mk::install should not be called directly. Most of the documentation would be for the defined types and the init calls. Lets keep that to another pull request.